### ISSUE TITLE
Give Players Wearing Infinity Helmet 100% Nutrition 

### ISSUE THREAD

**Issue by JustAlkaid** [GTNewHorizons/GT-New-Horizons-Modpack#14207]
### Your GTNH Discord Username

Alkaid

### Your Pack Version

2.6.1

### Your Proposal

As I said in the title.

### Your Goal

Since infinity helmet gives player a full bar of hunger, it is reasonable that it also can provide full nutrition. 

### Your Vision

Many players are not motivated to make infinity armors because the recipe is tedious. If this is added, it would be another motivation for players to make infinity armors.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Ilithios06** [comment]
Im not saying that this shouldnt be added, but I just want to say that by the time youre even close to being able to make the infinity armor set, most people will already have started using some other form of passive nutrition, primarily the personal healing nanobooster, which would make this addition somewhat redundant

**Comment by cainiaojun** [comment]
> Im not saying that this shouldnt be added, but I just want to say that by the time youre even close to being able to make the infinity armor set, most people will already have started using some other form of passive nutrition, primarily the personal healing nanobooster, which would make this addition somewhat redundant

Personal Healing NanoBooster doesn't give player any nutuition

**Comment by Rika-TH** [comment]
What would the benefits of 100% nutrition provide that just having pieces of the armour wouldn't? My view is that all it would serve to do is add more meaningless potion effects to your UI

**Comment by JustAlkaid** [comment]
I hope this can be added. I make a infinity armor production line and now they can only be used for Arcane Assembler. Why Infinite Armor can not be more useful?

**Comment by chochem** [comment]
If you are working on this, I think it should be 50% of course like other options that don't interact with the system.

**Comment by JustAlkaid** [comment]
> If you are working on this, I think it should be 50% of course like other options that don't interact with the system.

In fact I am trying to work on this but I still think it should be 100% because it is weird that an endgame armor would reduce your nutrition.

**Comment by joshaup1999** [comment]
Why does nutrition even matter? It’s just clutter of potions that provide little benefit

**Comment by JustAlkaid** [comment]
> Why does nutrition even matter? It’s just clutter of potions that provide little benefit

From my perspective the infinity armor heard kinda like something making me feel "I'm the king of the world" or something alike. So when I noticed it reduced my nutirtion I feel depressed that I still can't beat "nutrition" which annoyed me a lot in earlier stage even I am unkillable now.

Actually I don't think the infinity armor is a qualified endgame suit. It is so weak in flying, moving and other aspects. But I love the concept of "invulnerable"(and the apperance of infinity armor) so I wonder could I do something to improve it a little.

**Comment by EnderProyects** [comment]
Well since there are more tiers over infinity the inf armour right now is not in the endgame part...

**Comment by Ruling-0** [comment]
> Well since there are more tiers over infinity the inf armour right now is not in the endgame part...

It is by its nature of being the pinnacle of armor crafting, requiring basically every other end armor in its construction, yet is commonly considered weaker than one of the armors that crafts into it (draconic).

**Comment by miaowwwwww** [comment]
i have to say, i never considered crafting infi armor since it is just not better than draconic ones. We can make it better than draconic at least.

**Comment by JustAlkaid** [comment]
> i have to say, i never considered crafting infi armor(as armor) since it is just not better than draconic ones. We can make it better than draconic at least.

Yeah so we can start from adding nutrition

**Comment by ShadowOnWiiU** [comment]
I agree that this is a good idea. The nutrition benefits are (if i remember correctly) health boost and less knockback. Those are both two combat benefits, which both fit well onto the final armour tier in the game

**Comment by barsikus007** [comment]
and also it would be nice if Infinity Helmet had debuf remover ability like draconic one

right now I am using infinity helmet with other draconic parts (due to configurability), and only thing I lack in infinity helmet is debuf remover

**Comment by Miklebe** [comment]
Funnily enough debuff removing is tied to infinity chestplate.
Or maybe it's a side effect of becoming unkillable.

**Comment by chochem** [comment]
I just tried and it correctly sets everything to 50%. That is the value one gets for anything that fills up hunger without interacting with the system.
